### ISSUE TITLE
java.io.IOException: Corrupted records: storageLength=55,000 recordsLength=54,968 recordSize=32

### ISSUE THREAD

**Issue by khodemobin** [pestphp/pest-intellij#164]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Plugin version   | 1.0.3
| Pest version     | x.y.z
| IntelliJ version | PS-212.4746.100
| OS               | Linux

### Stacktrace
```
java.io.IOException: Corrupted records: storageLength=55,000 recordsLength=54,968 recordSize=32
	at com.intellij.util.io.storage.AbstractRecordsTable.getRecordsCount(AbstractRecordsTable.java:90)
	at com.intellij.util.io.storage.AbstractRecordsTable.createNewRecord(AbstractRecordsTable.java:74)
	at com.intellij.history.core.LocalHistoryStorage.lambda$createNextRecord$10(LocalHistoryStorage.java:87)
	at com.intellij.util.ConcurrencyUtil.withLock(ConcurrencyUtil.java:234)
	at com.intellij.util.io.storage.AbstractStorage.withWriteLock(AbstractStorage.java:430)
	at com.intellij.history.core.LocalHistoryStorage.createNextRecord(LocalHistoryStorage.java:85)
	at com.intellij.history.core.ChangeListStorageImpl.writeNextSet(ChangeListStorageImpl.java:208)
	at com.intellij.history.core.ChangeList.doEndChangeSet(ChangeList.java:85)
	at com.intellij.history.core.ChangeList.endChangeSet(ChangeList.java:73)
	at com.intellij.history.core.LocalHistoryFacade.endChangeSet(LocalHistoryFacade.java:56)
	at com.intellij.history.integration.LocalHistoryEventDispatcher.endChangeSet(LocalHistoryEventDispatcher.java:68)
	at com.intellij.history.integration.LocalHistoryEventDispatcher.fileCreated(LocalHistoryEventDispatcher.java:75)
	at com.intellij.history.integration.LocalHistoryEventDispatcher.handleAfterEvent(LocalHistoryEventDispatcher.java:196)
	at com.intellij.history.integration.LocalHistoryEventDispatcher.lambda$handleAfterEvents$1(LocalHistoryEventDispatcher.java:186)
	at com.intellij.history.integration.IdeaGateway.runWithVfsEventsDispatchContext(IdeaGateway.java:147)
	at com.intellij.history.integration.LocalHistoryEventDispatcher.handleAfterEvents(LocalHistoryEventDispatcher.java:184)
	at com.intellij.history.integration.LocalHistoryEventDispatcher$LocalHistoryBulkFileListener.after(LocalHistoryEventDispatcher.java:251)
	at com.intellij.util.messages.impl.MessageBusImpl.invokeMethod(MessageBusImpl.java:645)
	at com.intellij.util.messages.impl.MessageBusImpl.invokeListener(MessageBusImpl.java:620)
	at com.intellij.util.messages.impl.MessageBusImpl.deliverMessage(MessageBusImpl.java:417)
	at com.intellij.util.messages.impl.MessageBusImpl.pumpWaitingBuses(MessageBusImpl.java:386)
	at com.intellij.util.messages.impl.MessageBusImpl.pumpMessages(MessageBusImpl.java:372)
	at com.intellij.util.messages.impl.MessageBusImpl.access$200(MessageBusImpl.java:33)
	at com.intellij.util.messages.impl.MessageBusImpl$MessagePublisher.invoke(MessageBusImpl.java:172)
	at com.sun.proxy.$Proxy175.beforePsiChanged(Unknown Source)
	at com.intellij.psi.impl.PsiManagerImpl.beforeChange(PsiManagerImpl.java:421)
	at com.intellij.psi.impl.PsiManagerImpl.beforeChildAddition(PsiManagerImpl.java:197)
	at com.intellij.psi.impl.file.impl.PsiVFSListener.lambda$fileCreated$0(PsiVFSListener.java:171)
	at com.intellij.openapi.application.impl.ApplicationImpl.runWriteAction(ApplicationImpl.java:936)
	at com.intellij.psi.impl.file.impl.PsiVFSListener.fileCreated(PsiVFSListener.java:159)
	at com.intel
```

**Comment by olivernybroe** [comment]
Thanks for reporting, unfortunately the stack trace here has been cut off, please provide me with the full stacktrace.

**Comment by olivernybroe** [comment]
Issue has been fixed and will be released as part of v1.1.0.
You can find the latest release under releases tab for download manually or wait until the version has been approved in JetBrains Marketplace.
https://github.com/pestphp/pest-intellij/releases

Install it manually using
  <kbd>Preferences</kbd> > <kbd>Plugins</kbd> > <kbd>⚙️</kbd> > <kbd>Install plugin from disk...</kbd>
